pkg/services/store/service.go

```diff
@@ -49,9 +49,14 @@ type StorageService interface {
 	sanitizeUploadRequest(ctx context.Context, user *models.SignedInUser, req *UploadRequest, storagePath string) (*filestorage.UpsertFileCommand, error)
 }
 
+type storageServiceConfig struct {
+	allowUnsanitizedSvgUpload bool
```

Comment:
Is this option always false at the moment?

Reply:
yes -- we should only do this with an affirmative configuration